pimd/pim6_cmd.c

```diff
@@ -42,7 +42,94 @@
 #include "pimd/pim6_cmd_clippy.c"
 #endif
 
+DEFPY (ipv6_pim_rp,
+       ipv6_pim_rp_cmd,
+       "ipv6 pim rp X:X::X:X$rp [X:X::X:X/M]$group",
+       IPV6_STR
+       PIM_STR
+       "Rendevous Point\n"
```

Comment:
Rendezvous Point ?

Reply:
Thanks @rgirada for pointing it out. Have fixed it.